wordpress-mobile/WordPress-iOS
# Site Monitoring: Add PHP logs and web server logs

Closes #22460 
Closes #22461

## How to test
- Switch to an atomic site where you're an admin
- Go to more menu > site monitoring
- Verify php logs match Calypso
- Verify web server logs match Calypso

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
